idaes/core/reaction_base.py

```diff
@@ -31,7 +31,14 @@
                                     is_reaction_parameter_block,
                                     is_state_block)
 from idaes.core.util.misc import add_object_reference
-import idaes.core.util.scaling as iscale
+try:
+    # WHY on Python 3.6, this fails with "AttributeError: module 'idaes' has no attribute 'core'"
+    # this is likely due to a bug/limitation in how the Python import mechanism resolves circular imports
+    # for more information, see https://stackoverflow.com/questions/24807434
+    # and the official Python bug report: http://bugs.python.org/issue30024
+    import idaes.core.util.scaling as iscale
+except (AttributeError, ImportError):
+    from idaes.core.util import scaling as iscale
```

Comment:
As a more general solution, would it be better to just import scaling this way and be done with it?

I.e. Is there a benefit to trying the other way first with the `try/except`?

Reply:
That would work, too. My goal was to emphasize the fact that the original syntax caused an error, but the same could be achieved with a comment.